src/core/dfn.js

```diff
@@ -33,15 +33,17 @@ export function run(conf) {
 
     // add automatic pluralization to `data-lt` attributes
     // see https://github.com/w3c/respec/pull/1682
-    const normText = norm(dfn.textContent);
+    const normText = norm(dfn.textContent).toLowerCase();
     if (
-      conf.pluralize === true &&
+      pluralizeDfn &&
       !dfn.hasAttribute("data-lt-noPlural") &&
-      shouldPluralize(normText)
+      !dfn.hasAttribute("data-lt-noDefault")
```

Comment:
This is required as:
``` html
<dfn data-lt="baz" data-lt-noDefault>bar</dfn> can be referenced
        as <a>baz</a>
        but not as <a>bar</a> or <a>bars</a>
```

without `!dfn.hasAttribute("data-lt-noDefault")`, `links.has(plural) && !dfns.has(plural) === true`
Took me a while to realise this, so reporting :)